marklogic-client-api/src/main/java/com/marklogic/client/impl/OkHttpServices.java

```diff
@@ -5645,6 +5692,21 @@ public void writeTo(BufferedSink sink) throws IOException {
     }
   }
 
+  public static RequestBody makeRequestBodyForContent(AbstractWriteHandle content) {
```

Comment:
Extracted this from the Bulk DS code so it can be used for building a multipart/form-data request.